### PR TITLE
Backport #1801 (CUMULUS-2116), add audit updates from master

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2116](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2116)

Backports 2116 + audit updates to master dependencies

This change-set should map directly to #1801.    Reviewers can also consult #1817 as a reference PR - that PR was the full changeset from the original PR branch, which includes several TS migrations and an update to CMA-js that were pulled in during the work, but are disjoint from the resolution changeset.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,14 +76,14 @@
     "@typescript-eslint/eslint-plugin": "^2.29.0",
     "@typescript-eslint/parser": "^2.29.0",
     "audit-ci": "^2.4.2",
-    "ava": "^3.8.2",
+    "ava": "^3.10.1",
```
Comment (Jkovarik) [inline package.json:79]: These updates are bringing in audit fixes from master.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   rather than `null` if the mime type could not be found.
 - The `@cumulus/ingest/lock.removeLock` function now returns `undefined`
 
+### Added
```
Comment (markdboyd) [inline CHANGELOG.md:15]: Seems like a bad CHANGELOG merge

Comment (Jkovarik) [reply CHANGELOG.md:15]: I think it's actually a bad github diff - note this isn't the version update PR, just pulling 2116  into the branch.    That looks like master's changelog https://github.com/nasa/cumulus/blob/master/CHANGELOG.md in that the 2116 content is in 'added'

Comment (Jkovarik) [reply CHANGELOG.md:15]: Version up PR (based to this branch) https://github.com/nasa/cumulus/pull/1819

**Pull request merged**